config/config.go

```diff
@@ -44,7 +44,7 @@ type StateCommitConfig struct {
 	SnapshotWriterLimit int `mapstructure:"snapshot-writer-limit"`
 
 	// CacheSize defines the size of the cache for each memiavl store.
-	// defaults to 100000.
+	// Deprecated: this is removed, we will just rely on mmap page cache
```

Comment:
Why not just remove it altogether?

Reply:
Yeah good point, I can make another PR to clean it up, since we also need to remove it from cosmos repo.

Reply:
For now I'm cutting a tag with this change so that we can get it deployed to unblock evm work